src/main/java/com/limechain/trie/cache/node/PendingTrieNodeChange.java

```diff
@@ -0,0 +1,4 @@
+package com.limechain.trie.cache.node;
+
+public interface PendingTrieNodeChange {
+}
```

Comment:
Doesn't seem like a valuable interface

Reply:
Depends if we want to segregate updates from deletions in the cache. Initially I had the idea of doing so, but since we want to keep stale nodes for the prune trie we could remove the delete notion from the cache. @ablax  wdyt? Do we need to delete the stale nodes if they're no longer referenced in the main trie?

Reply:
No, we should keep at least 256 blocks trie history. Everything after the 256 block limit can be pruned.